### ISSUE TITLE
Questions with respect to the code.

### ISSUE THREAD

**Issue by d12306** [liyiying/Feature_Critic#1]
Hi, @liyiying , Thanks for your implementation. 

I am a little confused with the method in the paper. Actually, your aim in the paper is the domain generalization. However, in the target domain, we may not access any labels and the data. Therefore, I am wondering why you choose to train a SVM or KNN classifier in the paper? So in the testing stage, how will your model function? Please forgive me cause maybe I am asking a silly question

**Comment by liyiying** [comment]
For the heterogeneous DG, the label space of the target domain is totally different from the source domains'. So after trained on the source domains, we freeze the feature network and only use (some/K shots of) the training data in the target domain to train the specific classifiers. (For example, it's not possible to directly use a 1 to 10 digits classifier to classify the flowers, trees....). For homogeneous DG, the label space is the same among all domains, so no need to train a SVM/KNN then.

**Comment by d12306** [comment]
Thank you so much

**Comment by d12306** [comment]
@liyiying ,Hi, would you mind upload the code for other heteregenous baselines?like Reptile
